docs/pages/machine-workload-identity/machine-id/deployment/bound-keypair.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Deploying Machine ID with Bound Keypair Joining
+description: "How to install and configure Machine ID with Bound Keypair joining"
+---
+
+In this guide, you will install Machine ID's agent, `tbot`, on an arbitrary
+node. This node could be a bare-metal machine, a VM, a container, or any other
+host - the only requirement is that the node has persistent storage.
+
+Note that this is a generic join method. A platform-specific join method may be
+available that is better suited to your environment; refer to the
+[deployment guides](./deployment.mdx) for a full list of options.
+
+## How it works
+
+With Bound Keypair joining, instead of using a static token or relying solely on
+renewable certificates, Machine ID bots generate a unique keypair. Teleport is
+then configured to trust this public key for future joining attempts.
+
+Later, when the bot attempts to join the cluster, Teleport issues it a challenge
+that can only be completed using its private key. The bot returns the solved
+challenge, attesting to its own identity, and is conditionally allowed to join
+the cluster.
+
+As self attestation is inherently less secure than the external verification
+that would be provided by a cloud provider like AWS or a dedicated TPM, bound
+keypair joining enforces a number of additional checks to prevent abuse,
+including:
+- Join state verification to ensure the keypair cannot be usefully shared or
+  duplicated
+- Certificate generation counter checks to ensure regular bot certificates
+  cannot be usefully shared or duplicated
+- Configurable limits on how often - if at all - bots may be allowed to recover
```

Comment:
Do we expect readers to know what it means to "recover" at this point in the guide, or do we need to elaborate?